projects/1.16/assets/tinkers-construct/tconstruct/book/tinkers_gadgetry/zh_cn/armor/skull.json

```
@@ -0,0 +1,13 @@
+{
+  "tool": "tconstruct:slime_helmet",
+  "text": [
+    { "text": "我发现末影黏液可以用来模仿怪物的效果并且还有亮点，比如头颅" },
```

```suggestion
    { "text": "我发现只要有一个合适的触媒，比如头颅，就可以使用末影黏液来模仿怪物的效果。" },
```
这里原文稍微有点难懂，不过结合黏液头颅的制作方式，我倾向于认为这句话意思是末影黏液需要一个什么媒介（原文叫focus）才能模仿各种怪物（降低检测半径，还有一些额外特性），而这个媒介就是怪物头颅。